tsoa/sim.w

```diff
@@ -27,6 +28,7 @@ pub class Service_sim impl types.IService {
     }
 
     this.state = new sim.State();
+    this.bucket = new cloud.Bucket();
```

Comment:
Just curious (not sure it matters):
Why put it into a new bucket? Why not use the `sim.State`?